### PR TITLE
Match exactly and server settings updates

### LINKED ISSUE #1219: Update Collect server setup to match new server settings
https://github.com/getodk/collect/pull/3818 changes the layout and flow server settings in Collect. We should update any screenshots and descriptions to match this.

### LINKED ISSUE #1238: Add documentation for Match Exactly
Based on https://forum.getodk.org/t/have-collect-exactly-match-the-forms-on-central/23101

Changes we need to make to https://docs.getodk.org/collect-settings/#form-management-settings:

- [x] Screen shot
- [x] Add "Blank form update mode" section
- [x] Revise "Periodic form updates check"
- [x] Revise "Automatic download"

Change we need to make to https://docs.getodk.org/collect-import-export/#list-of-keys-for-all-settings:

- [x] Add form update mode key to JSON "schema"

### LINKED ISSUE #1254: Add documentation on Collect "soft delete"
Collect now "soft deletes" forms that still have filled forms. We should make sure it's clear what the ramifications of this are in the docs.

### PR DESCRIPTION
**Pull request by seadowg**
Closes #1238 
Closes #1219 
Closes #1254

#### What is included in this PR?

Just what's covered in the issue in terms of docs changes.

#### What new issues will need to be opened because of this PR?

None.

#### What is left to be done in the addressed issue?

Nothing.

#### What problems did you encounter?

Every time I go to make docs changes I find it hard to remember what command to run to get the auto build going as it's not using a tool with an idiomatic start command. To make this a little easier I added a `run.sh` (and `.bat`) and updated the `README`.

### CODE REVIEW

File: odk1-src/collect-settings.rst [hunk 1]
```diff
@@ -183,6 +183,18 @@ To access Form Management settings:
 
 .. rubric:: Form update
 
+:guilabel:`Blank form update mode`
+"""""""""""""""""""""""""""""""""""
+
+  Specifies how blank forms should be updated:
+
+  :guilabel:`Manually`
+    Enumerators will have to manually manage blank forms on the device using :guilabel:`Get Blank Form` and :guilabel:`Delete Saved Form`.
```
Comment (lognaturel) [inline odk1-src/collect-settings.rst:192]: `Enumerators will have to manually` -> `Enumerators manually`

File: odk1-src/collect-settings.rst [hunk 2]
```diff
@@ -183,6 +183,18 @@ To access Form Management settings:
...
+  :guilabel:`Previously downloaded forms only`
+    The enumerator will receive a notification when one or more forms already downloaded have an update available to their form definition or media files. Tapping on the notification will go to :guilabel:`Get Blank Form` where the user can choose to download some or all of the updated forms.
```
Comment (lognaturel) [inline odk1-src/collect-settings.rst:194]: `The enumerator` -> `Enumerators`

`forms already downloaded` -> `forms on the device`

File: odk1-src/collect-settings.rst [hunk 3]
```diff
@@ -183,6 +183,18 @@ To access Form Management settings:
...
+  :guilabel:`Exactly match server`
+    Collect will automatically download and update forms based on what's on the server. In addition, forms not on the server will be deleted from Collect. This mode hides :guilabel:`Get Blank Form` and the :guilabel:`Blank Forms` tab in :guilabel:`Delete Saved Form` as they are not required. The enumerator is still able to manually update forms with the server on the :guilabel:`Fill Blank Form` screen.
```
Comment (lognaturel) [inline odk1-src/collect-settings.rst:196]: `The enumerator is still able to manually update` makes it sound like they can pick forms. Maybe `The enumerator can trigger an update from the server on...`

**Comment by seadowg** [comment]
@lognaturel all good notes! I thought I'd used ImageOptim on the images but just checked and looks like I didn't realize you have to wait for the optimization process to finish 🙈. I'll look at the inline fixes and try and work out a way to incorporate the "who should use this" style docs.

**Review by lognaturel — DISMISSED** [review]

**Comment by seadowg** [comment]
Actually makes sense to do #1254 in here as well

File: odk1-src/collect-forms.rst [hunk 4]
```diff
@@ -203,7 +203,12 @@ Pulling Forms into Briefcase
 Deleting Forms
 ===============
 
-You can delete :formstate:`Blank` forms as well as filled forms in any state (:formstate:`Saved`, :formstate:`Finalized`, or :formstate:`Sent`). Deleting :formstate:`Blank` forms or filled forms in :formstate:`Saved` or :formstate:`Finalized` state removes every trace of that form definition or filled form. Deleting a :formstate:`Sent` form deletes the form contents but metadata associated with it including the deletion date and the instance name are maintained for display in the :guilabel:`View Sent Form` list.
+.. warning::
+    In Collect 1.28 filled instances of forms will still be editable after their blank form is deleted. This means that the form definition and media files will remain on the device until all the filled instances have been deleted.
```
Comment (lognaturel) [inline odk1-src/collect-forms.rst:207]: I think the warning here is actually about versions prior to v1.28 so I'd put that first. Then I think Collect v1.28+ is clearer. Or "Collect v1.28 and later".

File: odk1-src/collect-connect.rst [hunk 5]
```diff
@@ -20,9 +20,9 @@ Configure server from QR code
   :alt: Configure via QR code
   :class: device-screen-vertical
 
-1. Position the QR code in the center of the camera field, under the red line. When the camera focuses on the code, it will beep and scan the code.
+2. Position the QR code in the center of the camera field, under the red line. When the camera focuses on the code, it will beep and scan the code.
 
-1. Collect will apply the settings from the code and go back to the landing screen.
```
Comment (lognaturel) [inline odk1-src/collect-connect.rst:25]: rst will correctly update the numbers if you use 1 everywhere. This is preferable to numbering manually (same in markdown).

File: odk1-src/collect-connect.rst [hunk 6]
```diff
@@ -33,13 +33,34 @@ Configure server from QR code
...
+1. From the Action Button (:guilabel:`⋮`), select :menuselection:`General Settings`
+
+
+   .. image:: /img/collect-connect/main-menu-highlight-kebab.*
+     :alt: The Main Menu screen of the Collect app. The three-dot 'kebab' menu in the upper-right corner is circled in red.
+
+   .. image:: /img/collect-connect/kebab-menu-general-settings.*
+     :alt: The Main Menu screen of the Collect App. A modal menu has unrolled in the top-right corner, with the option *About*, *General Settings*, and *Admin Settings*. *General Settings* is circled in red.
+
+2. Select :guilabel:`Server`
```
Comment (lognaturel) [inline odk1-src/collect-connect.rst:48]: Use `1.` for each list item.

File: odk1-src/collect-connect.rst [hunk 7]
```diff
@@ -33,13 +33,34 @@ Configure server from QR code
 
 .. _other-collect-server-options:
 
+Configure server from settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```
Comment (lognaturel) [inline odk1-src/collect-connect.rst:38]: Maybe add an introductory sentence? "If you don't have a configuration QR code, you can manually enter your server URL and optional credentials."

File: odk1-src/collect-connect.rst [hunk 8]
```diff
@@ -33,13 +33,34 @@ Configure server from QR code
...
+     :alt: The Server Settings screen in the Collect app. Below the *Type* option there are three items labeled *URL*, *Username*, and *Password*.
+
 Other options
 ~~~~~~~~~~~~~~~
 
 .. toctree::
   :maxdepth: 1
 
   collect-connect-google
-  collect-connect-aggregate
```
Comment (lognaturel) [inline odk1-src/collect-connect.rst:43]: In `s3_website.yml`, add `collect-connect-aggregate/index.html: collect-connect`

**Review by lognaturel — CHANGES_REQUESTED** [review]

File: odk1-src/collect-forms.rst (hunk 4 above)
Comment (lognaturel) [reply odk1-src/collect-forms.rst:207]: Also note we use a `v` prefix consistently when referring to software versions.

File: odk1-src/collect-connect.rst (hunk 5 above)
Comment (seadowg) [reply odk1-src/collect-connect.rst:25]: This wasn't working on this page for some reason which is why I changed it. Can try again.

**Review by lognaturel — APPROVED** [review]

**Pull request merged**